onion-architecture/src/main/Main.java

```diff
@@ -0,0 +1,93 @@
+import application.TodoService;
+import domain.TodoItem;
+import infrastructure.TodoRepositoryImpl;
+
+import java.util.Scanner;
+
```

Comment:
Here above the `Main`, explain the pattern and describe how this code example implements it.